shared/polyfills/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@mux-elements/polyfills",
+  "version": "0.1.0",
+  "main": "./dist/index.js",
```

Comment:
since this is such a lightweight package, can we use `./src/index.ts`	as export and skip all the build scripts?
feels a bit heavy here

Reply:
I think if we're worried about ts vs. js, load, we can probably just migrate the polyfill to js. I'd recommend we do that as a followup since this feels like a lower priority.

Reply:
you're right yes, that makes the most sense. writing the polyfill in plain JS. no need for any extra build steps then

Reply:
if we're thinking of re-writing into plain JS. Thoughts about having the files at the root of the package only? Then import can be `@mux-elements/polyfills/window` without worrying about `dist/` directories etc.